CustomAnimations/Fade.py

```diff
@@ -5,12 +5,12 @@
 
 class Fade(BaseStripAnim):
     def __init__(self, layout, *, range=(255, 0), mask=(1, 1, 1), **kwds):
-        super().__init__(layout, preclear=False, **kwds)
```

Comment:
When this ran, you didn't see anything!

The reason is that the whole `color_list` isn't set up in the constructor.  In order to play with the `color_list`, you need to be in either 

* `pre_run` - a method that's called exactly once, before the first time `step()` is called, or
* `step`